slick/src/main/scala/slick/jdbc/MySQLProfile.scala

```diff
@@ -252,14 +252,12 @@ trait MySQLProfile extends JdbcProfile { profile =>
   }
 
   class UpsertBuilder(ins: Insert) extends super.UpsertBuilder(ins) {
+    @deprecated("Method kept for binary compatibility in 3.x, but should be removed in 3.4.x")
```

Comment:
```suggestion
    @deprecated("No longer used", "3.4.0")
```

Reply:
I think it needs to be removed in 4.0.0. The best way to track that is with a separate PR targeting the appropriate branch. (I guess it would have to be rebased once this is merged.)

@hvesalai @SethTisue what's the branching strategy to target 4.0.0 while 3.4.0 is in development?